server/server.go

```diff
@@ -241,6 +241,21 @@ func NewServer(ctx context.Context, opts ArgoCDServerOpts) *ArgoCDServer {
 	}
 }
 
+const (
+	notObjectErrMsg = "object does not implement the Object interfaces"
```

Comment:
Can you link to the issue for context, for the next person who is trying to figure out why this exists?

Reply:
sure, good idea!